install.sh

```diff
@@ -9,15 +9,15 @@ install_dir() {
 add_shorebird_to_path() {
   # Add the Shorebird CLI to your PATH.
   echo "Adding Shorebird to your PATH"
-  
+
   # Check if using zsh
   if [ -z "${ZSH_VERSION}" ]; then
-    echo "export PATH=\"$(install_dir)/bin:\$PATH\"" >> ~/.zshrc
-    zsh && source ~/.zshrc
+    echo "Updating ~/.zshrc"
+    echo "export PATH=\"$(install_dir)/bin:\$PATH\"" >>~/.zshrc
```

Comment:
Are you intentionally removing the space between >> and ~?  It looks like modern shells support ~, but I remember used to having to use $HOME for this sort of thing, presumably that's gone now.

Reply:
I'm using a formatter which removes the space so I assumed it didn't matter 🤷

Reply:
🤷